code/game/objects/items/devices/radio/headset.dm

```diff
@@ -81,13 +79,41 @@ GLOBAL_LIST_INIT(channel_tokens, list(
 	. = ..()
 	.["headset"] = TRUE
 
-/obj/item/radio/headset/syndicate //disguised to look like a normal headset for stealth ops
+/obj/item/radio/headset/alt
+	name = "bowman headset"
+	desc = "An updated, modular intercom that fits over the head. Protects ears from flashbangs."
+	icon_state = "headset_alt"
+	item_state = "headset_alt"
 
-/obj/item/radio/headset/syndicate/alt //undisguised bowman with flash protection
+/obj/item/radio/headset/alt/ComponentInitialize()
+	. = ..()
+	AddComponent(/datum/component/wearertargeting/earprotection, list(ITEM_SLOT_EARS))
+
+//syndicate
+/obj/item/radio/headset/syndicate
 	name = "syndicate headset"
-	desc = "A syndicate headset that can be used to hear all radio frequencies. Protects ears from flashbangs."
+	desc = "A headset worn by members of the various Syndicate splinters on the frontier."
 	icon_state = "syndie_headset"
 	item_state = "syndie_headset"
+	keyslot = new /obj/item/encryptionkey/syndicate
+
+/obj/item/radio/headset/syndicate/captain
+	name = "syndicate leader headset"
+	desc = "A headset worn by officers of the various Syndicate splinters on the frontier. Protects ears from flashbangs."
+	command = TRUE
+	keyslot2 = new /obj/item/encryptionkey/heads/captain
+
+/obj/item/radio/headset/syndicate/alt
+	name = "syndicate bowman headset"
+	desc = "A headset worn by members of the various Syndicate splinters on the frontier. Protects ears from flashbangs."
+	icon_state = "syndie_headset_alt"
+	item_state = "syndie_headset_alt"
+
+/obj/item/radio/headset/syndicate/alt/captain
+	name = "syndicate leader bowman headset"
+	desc = "A headset worn by officers of the various Syndicate splinters on the frontier. Protects ears from flashbangs."
+	command = TRUE
+	keyslot2 = new /obj/item/encryptionkey/heads/captain
```

Comment:
Do these actually have ear protection? The one above at line 88 doesn't seem to be for the syndicate one

Reply:
They won't have ear protection unless they put this in their Initialize proc
```
AddComponent(/datum/component/wearertargeting/earprotection, list(ITEM_SLOT_EARS))
```

Reply:
I actually left the ear protection blurb in the description on accident, it's fixed in the latest commit.